src/components/Common/ConfirmDialog.tsx

```diff
@@ -1,6 +1,21 @@
-import { ButtonVariant, Cancel, Submit } from "@/components/Common/ButtonV2";
+import { Button } from "@/components/ui/button";
+
 import DialogModal from "@/components/Common/Dialog";
 
+type ButtonVariant =
+  | "default"
+  | "destructive"
+  | "outline"
+  | "primary"
+  | "secondary"
+  | "ghost"
+  | "link"
+  | "outline_primary"
+  | "primary_gradient"
+  | "white"
+  | "warning"
+  | "alert";
+
```

Comment:
let's not redefine button variants here. here the situation is different from callout since callout's variant prop defined how the callout would look like. but here the variant prop directly is passed to the submit button's prop, which is "ButtonVariant" itself. import it instead